packages/matchbox/src/components/ActionList/styles.ts

```diff
@@ -20,7 +20,7 @@ export const StyledSection = styled(Box)`
 `;
 
 export const StyledLink = styled(UnstyledLink)`
-  ${props => (props.isType ? buttonReset : '')}
+  ${(props) => (props.isType ? buttonReset : '')}
```

Comment:
May want to back out some of these style changes because they might conflict with the other pr